lib/models/follow_up.dart

```diff
@@ -0,0 +1,159 @@
+    DateTime? updatedAt,
+  }) : super(objectId: objectId, createdAt: createdAt, updatedAt: updatedAt);
+
+  /// denotes the distance.
+  final int? distance;
+
+  /// denotes the bloodDiastolic.
+  final int? bloodDiastolic;
+
+  /// denotes the bloodSystolic.
+  final int? bloodSystolic;
+
+  /// denotes the rythmus.
+  final String? rhythm;
+
+  /// denotes the rythmusTyp.
+  final String? rhythmTyp;
+
+  /// denotes the testResult.
+  final int? testResult;
+
+  /// denotes the healthState.
+  final List<dynamic>? healthState;
+
+  /// denotes the locationType.
+  final String? electricalAxisDeviation;
+
+  /// denotes the heartRate.
+  final int? heartRate;
+
+  /// denotes the healthScore.
+  final int? healthScore;
+
+  /// denotes the number.
+  final int? number;
+
+  /// denotes the doctor's object ID.
+  final String doctorObjectId;
+
+  /// denotes the patient's object ID.
+  final String patientObjectId;
+
+  /// The database table the objects are stored in.
+  static const String databaseTable = 'followUP';
+
+  @override
+  get table {
+    return databaseTable;
+  }
+
+  /// Returns a copy of this follow up with the given values updated.
+  @override
+  FollowUp copyWith({
+    int? distance,
+    int? bloodDiastolic,
+    int? bloodSystolic,
+    String? rythmus,
+    String? rythmusTyp,
+    int? testResult,
+    List<dynamic>? healthState,
+    String? locationType,
+    int? heartRate,
+    int? healthScore,
+    int? number,
+    String? patientObjectId,
+    String? doctorObjectId,
+    String? objectId,
+    DateTime? createdAt,
+    DateTime? updatedAt,
+  }) {
+    return FollowUp(
+      distance: distance ?? this.distance,
+      bloodDiastolic: bloodDiastolic ?? this.bloodDiastolic,
+      bloodSystolic: bloodSystolic ?? this.bloodSystolic,
+      rhythm: rythmus ?? this.rhythm,
+      rhythmTyp: rythmusTyp ?? this.rhythmTyp,
+      testResult: testResult ?? this.testResult,
+      healthState: healthState ?? this.healthState,
+      electricalAxisDeviation: locationType ?? this.electricalAxisDeviation,
+      heartRate: heartRate ?? this.heartRate,
+      healthScore: healthScore ?? this.healthScore,
+      number: number ?? this.number,
+      patientObjectId: patientObjectId ?? this.patientObjectId,
+      doctorObjectId: doctorObjectId ?? this.doctorObjectId,
+      objectId: objectId ?? this.objectId,
+      createdAt: createdAt ?? this.createdAt,
+      updatedAt: updatedAt ?? this.updatedAt,
+    );
+  }
+
+  @override
+  List<Object> get props => <Object>[
+        patientObjectId,
+        doctorObjectId,
+      ];
+
+  @override
+  Map<String, dynamic> get databaseMapping => <String, dynamic>{
+        if (distance != null) 'Strecke': distance,
+        if (bloodDiastolic != null) 'BD_Diastolisch': bloodDiastolic,
+        if (bloodSystolic != null) 'BD_Systolisch': bloodSystolic,
+        if (rhythm != null) 'Rythmus': rhythm,
+        if (rhythmTyp != null) 'RythmusTyp': rhythmTyp,
```

Comment:
'Typ' in 'RythmusTyp' should be written in lowercase and an 'h' is missing

Reply:
i do not think we have to write RythmusTyp in lowercase:
In DB:
![image](https://github.com/healthcare-it-solutions/picos/assets/132355030/d70ef5ff-0386-4fde-a91d-58a4a131f0e4)

Reply:
Of course, you are right!